docs/REST API/rest-api.md

```diff
@@ -8926,13 +8936,26 @@ Returns information about the specified software. By default, `versions` are sor
       "installer_id": 23,
       "team_id": 3,
       "uploaded_at": "2024-04-01T14:22:58Z",
+      "install": "automatic"
       "install_script": "sudo installer -pkg /temp/FalconSensor-6.44.pkg -target /",
       "pre_install_query": "SELECT 1 FROM macos_profiles WHERE uuid='c9f4f0d5-8426-4eb8-b61b-27c543c9d3db';",
       "post_install_script": "sudo /Applications/Falcon.app/Contents/Resources/falconctl license 0123456789ABCDEFGHIJKLMNOPQRSTUV-WX",
       "self_service": true,
+      "labels_exclude_any": [
+        {
+          "name": "Label name 1",
+          "id": 1
+        },
+        {
+          "name": "Label name 2",
+          "id": 2
+        }
+      ],
       "status": {
```

Comment:
### Dev note

We can break this API endpoint by removing the `"installed"` status because app management features are tagged as experimental: https://github.com/fleetdm/fleet/pull/21164/files

More on experimental features in the handbook here: https://fleetdm.com/handbook/company/product-groups#experimental-features